inst/EpiModel2.rmd

```diff
@@ -79,6 +86,7 @@ Along with the change to how we build two-group networks, we have also changed t
 Input parameters for each of these two functions, for the second group, have also changed with EpiModel moving from mode to group language. Second group parameters or initial conditions are now denoted by a `.g2` suffix as opposed to the `.m2` suffix that was used previously:
```

Comment:
Suggested changes/comments for line 80 below - question again about `group` vs. "group".

The paragraph below explaining the table is a little confusing. I am not sure what the difference is between the last two scenarios. Do you mean if the attribute is present in the formation model but not on the network for the last scenario? 

> As shown above, if an attribute (in this case `group`) is on the network and both present and absent in the formation model, a valid network can be estimated. If an attribute is missing from both the network and the formation model, a valid network will still result. If, however, an attribute is not present on the network and not in the formation model, this will not lead to a valid network as we cannot estimate edge formation with attributes not present in the networks attribute list.

Suggested changes for line 84 below:

> ## Base Function Changes
> Along with the change to how we build two-group networks, we have also changed the built-in EpiModel functions to make the workflow clearer (particularly for module development on the part of the user). In EpiModel 1.x, no distinction in module functionality was made for bipartite and non-bipartite networks. With EpiModel 2.0, we have split each of the appropriate core modules into one- and two-group sub-functions. Assignment of the correction function is done automatically by EpiModel based on the input parameters passed through `init.net` or `param.net`, if `control.net` input parameter `type` is specified (see next section). 
 
```suggestion
Input parameters for these two functions for the second group have also changed with EpiModel, moving from mode to group language. Second group parameters or initial conditions are now denoted by a `.g2` suffix as opposed to the `.m2` suffix that was used previously:
```